### PR TITLE
Tag LCIO v0.4.2 [https://github.com/jstrube/LCIO.jl]

### PR DESCRIPTION
**Pull request by jstrube**

### CODE REVIEW

File: LCIO/versions/0.5.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+BinDeps
+Compat
```
Comment (tkelman) [inline LCIO/versions/0.5.0/requires]: should declare a minimum version dependency that implements the features you need

File: LCIO/versions/0.5.1/requires [hunk 2]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+BinDeps 0.4.5
+Compat 0.9.2
```
Comment (tkelman) [inline LCIO/versions/0.5.1/requires]: you're using something that was only implemented in the most recent version?

Comment (jstrube) [inline LCIO/versions/0.5.1/requires]: I don't have any users right now and don't want to spend time figuring out how far I can go back. If there's a user request to support an older setup, I'll look into it.

Comment (tkelman) [inline LCIO/versions/0.5.1/requires]: there are specific instructions on the compat readme, it's a few clicks

**Comment by jstrube** [comment]
Thanks for merging. I appreciate the pointers, too

**Pull request merged**